### PR TITLE
Dịch "sentiment-analysis-cnn_vn" - Phần 5

### LINKED ISSUE #2601: Dịch "sentiment-analysis-cnn_vn" - Phần 5
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_natural-language-processing-applications/sentiment-analysis-cnn_vn.md#L364-L428

### Hướng dẫn
- [Dịch trực tiếp trên website](https://github.com/aivivn/d2l-vn/blob/master/CONTRIBUTING.md).
- Sau khi nhấn vào phần cần dịch ở trên, hãy chọn **Normal view** và làm theo hướng dẫn ở trên để có thể bắt đầu dịch.

### PR DESCRIPTION
**Pull request by LongNguyen1984**
Dịch lần 1

## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Điền số issue sau dấu `#` (không có dấu cách): Close #2601

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy thảo luận thêm ở thẻ Discussions của dự án.

* [x] Format của bản gốc có được giữ nguyên không?
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn. Bạn đã kiểm tra rằng mỗi đoạn văn đã nằm một dòng?

* [ ] Điền tên đầy đủ của bạn vào mục **Những người thực hiện** ở cuối tập tin.

## Lưu ý cho người phản biện (Reviewer)

1. Chỉ review khi Pull Request (PR) này đã sẵn sàng cho việc review. Một PR sẵn sàng khi PR đó không có label `status: wip`.
2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

### CODE REVIEW

File: chapter_natural-language-processing-applications/sentiment-analysis-cnn_vn.md [hunk 1]
```diff
@@ -402,7 +402,7 @@ d2l.train_ch13(net, train_iter, test_iter, loss, trainer, num_epochs, devices)
 Below, we use the trained model to classify sentiments of two simple sentences.
 -->
 
-*dịch đoạn phía trên*
+Dưới đây, ta sử dụng mô hình đã được huấn luyện để phân loại cảm xúc từ hai câu đơn giản.
```
Comment (cuongvng) [inline chapter_natural-language-processing-applications/sentiment-analysis-cnn_vn.md:405]:
```suggestion
Dưới đây, ta sử dụng mô hình đã được huấn luyện để phân loại cảm xúc của hai câu đơn giản.
```

File: chapter_natural-language-processing-applications/sentiment-analysis-cnn_vn.md [hunk 2]
```diff
@@ -422,7 +422,10 @@ d2l.predict_sentiment(net, vocab, 'this movie is so bad')
 * TextCNN mainly uses a one-dimensional convolutional layer and max-over-time pooling layer.
 -->
 
-*dịch đoạn phía trên*
+* Ta có thể dùng tích chập một chiều để xử lý và phân tích dữ liệu theo thời gian.
+* Một phép tương quan chéo một chiều với các kênh nhiều đầu vào có thể xem như một phép tương quan chéo hai chiều với một kênh một đầu vào.
```
Comment (cuongvng) [inline chapter_natural-language-processing-applications/sentiment-analysis-cnn_vn.md:426]:
```suggestion
* Phép tương quan chéo một chiều đa kênh đầu vào có thể xem như phép tương quan chéo hai chiều đơn kênh đầu vào.
```

File: chapter_natural-language-processing-applications/sentiment-analysis-cnn_vn.md [hunk 3]
```diff
@@ -422,7 +422,10 @@ d2l.predict_sentiment(net, vocab, 'this movie is so bad')
...
+* Đầu vào của lớp gộp cực đại theo thời gian có thể có số bước thời gian trên mỗi kênh khác nhau.
```
Comment (cuongvng) [inline chapter_natural-language-processing-applications/sentiment-analysis-cnn_vn.md:427]:
```suggestion
* Đầu vào của tầng gộp cực đại theo thời gian có thể có số bước thời gian trên mỗi kênh khác nhau.
```

File: chapter_natural-language-processing-applications/sentiment-analysis-cnn_vn.md [hunk 4]
```diff
@@ -435,8 +438,10 @@ tuning hyperparameters, using larger pre-trained word vectors, and using the spa
 3. What other natural language processing tasks can you use textCNN for?
 -->
 
-*dịch đoạn phía trên*
-
+1. Điều chỉnh các tham số mô hình và so sánh các phương pháp phân tích hai cảm xúc, sử dụng các mạng nơ rôn hồi quy và các mạng nơ rôn chập, ở khía cạnh độ chính xác và hiệu xuất tính toán.
```
Comment (cuongvng) [inline chapter_natural-language-processing-applications/sentiment-analysis-cnn_vn.md:441]:
```suggestion
1. Điều chỉnh các tham số mô hình và so sánh hai phương pháp phân tích hai cảm xúc, sử dụng các mạng nơ-ron truy hồi và các mạng nơ-ron tích chập, ở khía cạnh độ chính xác và hiệu suất tính toán.
```

File: chapter_natural-language-processing-applications/sentiment-analysis-cnn_vn.md [hunk 5]
```diff
@@ -435,8 +438,10 @@ tuning hyperparameters, using larger pre-trained word vectors, and using the spa
...
+2. Bạn có thể cải thiện thêm độ chính xác của mô hình trên tập kiểm tra thông qua việc sử dụng ba phương pháp đã được giới thiệu ở phần trước:
+điều chỉnh các tham số mô hình, sử dụng các vector từ tiền huấn luyện lớn hơn. và sử dụng công cụ token hóa từ spaCy.
```
Comment (cuongvng) [inline chapter_natural-language-processing-applications/sentiment-analysis-cnn_vn.md:443]:
```suggestion
điều chỉnh các tham số mô hình, sử dụng các vector từ tiền huấn luyện lớn hơn, và sử dụng công cụ token hóa từ spaCy.
```

**Review by cuongvng — COMMENTED** [review]
Thank anh!

**Review by cuongvng — APPROVED** [review]

**Review by duythanhvn — APPROVED** [review]
